pkg/localmetrics/localmetrics.go

```diff
@@ -56,6 +60,14 @@ var (
 		Help:        "The duration it takes to reconcile a ClusterDeployment",
 		ConstLabels: prometheus.Labels{"name": "certman-operator"},
 	})
+	MetricCertRequestsCount = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "certman_operator_certificate_requests_count",
+		Help: "Report the current count of Certificate Requests",
+	}, []string{"name"})
+	MetricCertIssuanceRate = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "certman_operator_certificate_issuance_rate",
```

Comment:
This looks more like a counter than a gauge because the value only goes up. Also rate shouldn't be part of the name because the rate is calculated by the query later.

Reply:
Agreed on the type. Regarding the `rate` in the name of the metric, it is because it should not be used as a counter (as never initialized, it does not represent an actual number of certificate). 
`certman_operator_issued_certificates_counter` would be better ?

Reply:
It doesn't need to be initialized because it will be zero by when created. As for the name `certman_operator_issued_certficates_count` would be in line with naming conventions used in most places.